src/training/CMakeLists.txt

```diff
@@ -76,6 +76,22 @@ set(common_training_src
     commandlineflags.h
     commontraining.cpp
     commontraining.h
+    ctc.cpp
+    ctc.h
+    errorcounter.cpp
+    errorcounter.h
+    intfeaturedist.cpp
+    intfeaturedist.h
+    intfeaturemap.cpp
+    intfeaturemap.h
+    mastertrainer.cpp
+    mastertrainer.h
+    networkbuilder.cpp
+    networkbuilder.h
+    sampleiterator.cpp
+    sampleiterator.h
+    trainingsampleset.cpp
+    trainingsampleset.h
 )
```

Comment:
@egorpugin, please review my changes for CMake. I don't know why there are three libraries for training and added the moved code to one of them:

    src/training/libunicharset_training.a
    src/training/libtessopt.a
    src/training/libcommon_training.a

Wouldn't it be simpler to have a single library for training?

Reply:
> Wouldn't it be simpler to have a single library for training?

Only in case when files are used in >1 target.
Otherwise personally I prefer to keep files near users (when use count ==1).